pages/glossary/disabled.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Disabled
+key: disabled
+---
+
+An element that is normally [perceivable][] and [operable][], but has been rendered [inoperable][operable] either temporarily or permanently using one of the following techniques, is considered disabled:
+
+1. The [`disabled`][disabled] attribute. The presence of this attribute, regardless of its value, either
+
+    - on a [`button`][button], [`input`][input], [`select`][select], [`textarea`][textarea], or [form-associated custom element][]; or
+    - on a [`fieldset`][fieldset] element
```

Comment:
I don't think it's worth having a two items list when the first item is itself 5 items and the exception caused by the second is anyway made explicit in the text immediately after.